csrc/scheduler/mma_utils.cpp

```diff
@@ -1986,17 +2089,15 @@ DimRolesMap MatmulPattern::getDimRoles(IdModel& id_model) const {
   // for each valgroup, store a pair of flags. The first records whether the
   // group is present at all in the tv. The second records whether the value is
   // concrete (i.e. not reduction, broadcast, or device).
-  std::unordered_map<ValGroup, std::pair<DimPresence, DimPresence>> flags;
+  std::unordered_map<ValGroup, DimPresence> flags;
```

Comment:
We never actually used the `present_flag` so I removed it.

Reply:
OK it seems that changes here actually caused some failures on Ampere. Looking into it now.

Reply:
Fixed in latest push.